mappings/net/minecraft/datafixer/fix/RenameSwapHandsFix.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_5404 net/minecraft/datafixer/fix/RenameSwapHandsFix
```

Comment:
Please remove this one. This is more like Rename options fix as well.

Reply:
Also #1523 covers it already